src/interface/meta.thrift

```diff
@@ -172,6 +172,56 @@ struct DropSpaceReq {
     2: bool if_exists,
 }
 
+enum AdminJobOp {
+    ADD         = 0x01,
+    SHOW_All    = 0x02,
+    SHOW        = 0x03,
+    STOP        = 0x04,
+    BACKUP      = 0x05,
+    RECOVER     = 0x06,
+    INVALID     = 0xFF,
+} (cpp.enum_strict)
+
+struct AdminJobReq {
+    1: AdminJobOp   op
+    2: list<string> paras;
+}
+
+struct JobDetails {
+    1: string id
+    2: string typeAndParas
+    3: string status
+    4: string startTime
+    5: string stopTime
+}
+
+struct TaskDetails {
+    1: string id
+    2: string host
+    3: string status
+    4: string startTime
+    5: string stopTime
+}
+
+struct BackupJobResult {
+    1: i32 jobNum
+    2: i32 taskNum
+}
+
+struct AdminJobResult {
+    1: optional i32                 jobId
+    2: optional list<JobDetails>    jobDetails
```

Comment:
Add comments for each field,  when the filed will be used. 
If i understand correctly, not all fields are filled for different requests.